testing/test_model.py

```diff
@@ -52,5 +52,35 @@ def test_film_generator():
     assert (type(inf[0]) == torch.Tensor)
     assert (type(inf[1]) == torch.nn.parameter.Parameter)
 
+def test_modified_Encoder():
```

Comment:
functions are in lower case according to [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
You're right, I will follow and revise it into the correct form.